scripts/github_action.py

```diff
@@ -0,0 +1,61 @@
+class GithubActionsTransformer():
+    def __init__(self, name):
+        super(GithubActionsTransformer, self).__init__()
+        self.actions = {
+            'on': [
+                "push",
+                "pull_request",
+                "pull_request_target",
+            ],
+            "jobs": {
```

Comment:
When I run this, I get the cmake jobs in a different order. Do we need to sort the job keys when writing out `cmake.yml` to avoid churn between python versions, or did you change something and forget to update the output?

Reply:
Good idea, let me add sort

Reply:
fixed.